### ISSUE TITLE
Postgis on autocomplete

### ISSUE THREAD

**Issue by jsanz** [CartoDB/franchise#19]
It'd be great to have postgis functions in the autocomplete catalog to help users to write queries.

**Comment by alrocar** [comment]
Note: Solve this by adding support to PostGIS in [codemirror](https://codemirror.net/mode/index.html), so we can benefit from BUILDER as well

**Comment by jsanz** [comment]
yes, maybe we can extend the `x-plsql` mode to something like `x-pgostgis` [here](https://github.com/codemirror/CodeMirror/blob/282cf36664d05f776eba88a1f8ace03c1f085cd0/mode/sql/sql.js#L378) worth checking!
